spec/decorators/service_provider_session_decorator_spec.rb

```diff
@@ -92,4 +69,27 @@
       expect(subject.sp_name).to_not be_nil
     end
   end
+
+  describe '#sp_logo' do
+    context 'service provider has a logo' do
+      it 'returns the logo' do
+        sp_logo = 'real_logo.svg'
+        sp = build_stubbed(:service_provider, logo: sp_logo)
+
+        subject = ServiceProviderSessionDecorator.new(sp: sp, view_context: view_context)
+
+        expect(subject.sp_logo).to eq sp_logo
+      end
+    end
+
+    context 'service provider does not have a logo' do
+      it 'returns the default logo' do
+        sp = build_stubbed(:service_provider, logo: nil)
+
+        subject = ServiceProviderSessionDecorator.new(sp: sp, view_context: view_context)
+
+        expect(subject.sp_logo).to eq ServiceProviderSessionDecorator::DEFAULT_LOGO
```

Comment:
Sure I can see that POV. 

One reason to ref the constant in the test is that it would mean we could change the file name / path that the constant points to and we wouldn't need to change the test (much like we can change copy without changing specs that look for localized copy strings because the keys stay the same).

Anyway, updated to look for string at f0c6338 👘